### ISSUE TITLE
Compiler + Xcode 10.1 Error

### ISSUE THREAD

**Issue by livemath** [devernay/xcodelegacy#49]
When I install the older compilers for Xcode 10.1 on High Sierra I get the following error at (any) compile:

: could not create session: requestError(description: "unknown error
: while handling message: unableToInitializeCore(errors:
: [\"/Applications/Xcode.app/Contents/PlugIns/Xcode3Core.ideplugin/
: Contents/SharedSupport/Developer/Library/Xcode/Plug-ins/GCC
: 4.0.xcplugin/Contents/Resources/GCC Generic.xcspec: warning: spec
: \\\':com.apple.compilers.gcc\\\' already registered from
: /Applications/Xcode.app/Contents/PlugIns/Xcode3Core.ideplugin/Contents
: /SharedSupport/Developer/Library/Xcode/Plug-ins/Clang LLVM
: 1.0.xcplugin/Contents/Resources/Default Compiler.xcspec\",
: \"/Applications/Xcode.app/Contents/PlugIns/Xcode3Core.ideplugin/
: Contents/SharedSupport/Developer/Library/Xcode/Plug-ins/GCC
: 4.2.xcplugin/Contents/Resources/GCC 4.2.xcspec: error: unknown spec
: \\\'Class\\\': \\\'PBXCompilerSpecificationGcc3_3\\\'\",
: \"/Applications/Xcode.app/Contents/PlugIns/Xcode3Core.ideplugin/
: Contents/SharedSupport/Developer/Library/Xcode/Plug-ins/GCC
: 4.0.xcplugin/Contents/Resources/GCC 4.0.xcspec: error: unknown spec
: \\\'Class\\\': \\\'PBXCompilerSpecificationGcc3_3\\\'\"])")

**Comment by ShikiSuen** [comment]
I have met exactly the same problem.

**Comment by ShikiSuen** [comment]
![image](https://user-images.githubusercontent.com/3164826/53423457-5620db80-3a1c-11e9-8cf2-79e533c46a8d.png)

**Comment by a2sheppy** [comment]
Same issue here. I spent a little time trying to figure it out and came up with very little, other than wondering if perhaps the new build system in Xcode 10.1 is a factor.

**Comment by gghose** [comment]
Yes, switch to the old build system, and things should [work](https://stackoverflow.com/questions/53050108/xcode-10-how-to-switch-to-old-build-system).
